docs/source/robuststats.rst

```diff
@@ -1,10 +1,9 @@
-Spatial statistics
+Robust statistics
 ==================
 
-How does one calculate the error between two DEMs?
-This question is the basis of numerous academic papers, but their conclusions are often hard to grasp as the mathematics behind them can be quite daunting.
-In addition, the lack of a simple implementation in a modern programming language makes these methods obscure and used only by those who can program it themselves.
-One of the goals of ``xdem`` is to simplify state-of-the-art statistical measures, to allow accurate DEM comparisons for everyone, regardless of one's statistical talent.
+Digital Elevation Models often contain outliers that hamper further analysis.
+In order to deal with outliers, ``xdem`` integrates statistical measures robust to outliers to be used for estimation of the
+mean or dispersion of a sample, or more complex function fitting.
```

Comment:
I cannot comment unchanged lines, but the definition of median (l 28) is wrong. Worth changing in this PR!